src/current/_includes/v23.2/sql/mixed-isolation-levels.md

```diff
@@ -0,0 +1,12 @@
+{% if page.name == "transactions.md" %}### Mixed isolation levels{% else if page.name == "transaction-layer.md" %}#### Mixed isolation levels{% endif %}
+
+Regardless of the isolation levels of other transactions, transactions behave according to their respective isolation levels: Statements in `SERIALIZABLE` transactions see data that committed before the transaction began, whereas statements in `READ COMMITTED` transactions see data that committed before each **statement** began. Therefore:
+
+- If a `READ COMMITTED` transaction `R` commits before a `SERIALIZABLE` transaction `S`, every statement in `S` will observe all writes from `R`. Otherwise, `S` will not observe any writes from `R`.
+- If a `SERIALIZABLE` transaction `S` commits before a `READ COMMITTED` transaction `R`, every **subsequent** statement in `R` will observe all writes from `S`. Otherwise, `R` will not observe any writes from `S`.
```

Comment:
formatting nit, add break tags
```suggestion
- If a `READ COMMITTED` transaction `R` commits before a `SERIALIZABLE` transaction `S`,<br>every statement in `S` will observe all writes from `R`.<br>Otherwise, `S` will not observe any writes from `R`.
- If a `SERIALIZABLE` transaction `S` commits before a `READ COMMITTED` transaction `R`,<br>every **subsequent** statement in `R` will observe all writes from `S`.<br> Otherwise, `R` will not observe any writes from `S`.
```
so it is a bit more readable

<img width="874" alt="Screenshot 2024-09-27 at 10 59 02 AM" src="https://github.com/user-attachments/assets/cd843f8f-924f-4ae2-bf45-975d4b33bad5">